docker/Dockerfile

```diff
@@ -1,41 +1,75 @@
-FROM registry.service.dsd.io/ruby:2.6.3
+FROM ruby:2.6.3-alpine3.8
+MAINTAINER Ministry of Justice <tools@digital.justice.gov.uk>
```

Comment:
Should this not be crowncourtdefence@digital.justice.gov.uk?
The tools team do not maintain the CCCD Dockerfile.

Reply:
@colinbruce do we monitor that one, in which case yes 👍

Reply:
We do, or at least, _I_ do! :)

Reply:
Cool, thats done